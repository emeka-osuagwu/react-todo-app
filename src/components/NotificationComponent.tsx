import React from "react";
import * as RB from "react-bootstrap";

function Component() {
    return (
        <RB.Alert variant={"danger"}>
            No result for the given search parameters!
        </RB.Alert>
    );
}

export default Component;
