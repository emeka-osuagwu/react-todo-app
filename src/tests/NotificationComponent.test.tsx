import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";

import "@testing-library/jest-dom/extend-expect";

import NotificationComponent from "../components/NotificationComponent";

test("renders without crashing", () => {
    const { getByText } = render(<NotificationComponent />);
    const title = getByText("No result for the given search parameters!");
    expect(title).toBeInTheDocument();
});

test("renders same mock up", () => {
    const tree = renderer.create(<NotificationComponent />).toJSON();
    expect(tree).toMatchSnapshot();
});
