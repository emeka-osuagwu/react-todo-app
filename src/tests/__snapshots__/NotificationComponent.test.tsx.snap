// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders same mock up 1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
  No result for the given search parameters!
</div>
`;
